tests/scadnano_tests.py

```diff
@@ -12,6 +13,7 @@
 import scadnano.origami_rectangle as rect
 import scadnano.modifications as mod
 
+from scadnano.scadnano import _convert_design_to_oxdna_system

```

Comment:
This seems like a cleaner way of testing equality to the oxDNA design.  The oxDNA tests already test that export format.